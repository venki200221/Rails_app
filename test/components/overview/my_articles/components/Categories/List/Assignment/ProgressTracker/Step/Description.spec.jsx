import React from 'react';
import { shallow } from 'enzyme';
import '../../../../../../../../../testHelper';

import Description from '../../../../../../../../../../app/assets/javascripts/components/overview/my_articles/components/Categories/List/Assignment/ProgressTracker/Step/Description';

describe('Description', () => {
  it('displays the content', () => {
    const component = shallow(<Description content="content" />);
    expect(component).toMatchSnapshot();
  });
});
