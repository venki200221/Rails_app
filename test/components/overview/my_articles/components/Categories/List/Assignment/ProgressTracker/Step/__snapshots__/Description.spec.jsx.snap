// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description displays the content 1`] = `ShallowWrapper {}`;
